src/ess/reduce/nexus/workflow.py

```diff
@@ -683,23 +683,21 @@ def LoadMonitorWorkflow(
     wf = sciline.Pipeline(
         (*_common_providers, *_monitor_providers),
         constraints=_gather_constraints(
-            run_types=run_types, monitor_types=monitor_types
+            run_types=run_types, monitor_types=monitor_types, component_types=[]
         ),
     )
     wf[PreopenNeXusFile] = PreopenNeXusFile(False)
     return wf
 
 
 def LoadDetectorWorkflow(
-    *,
-    run_types: Iterable[sciline.typing.Key],
-    monitor_types: Iterable[sciline.typing.Key],
```

Comment:
Not sure why we needed to have the monitor_types here, when we are loading detectors?
So I removed it.